src/de/muenchen/allg/itd51/wollmux/sidebar/WollMuxSidebarFactory.java

```diff
@@ -34,6 +36,7 @@
   public WollMuxSidebarFactory(XComponentContext context)
   {
     LOGGER.debug("WollMuxSidebarFactory:WollMuxSidebarFactory");
+    WollMuxEventHandler.getInstance().handleInitialize();
```

Comment:
Hier sollte der WollMux doch schon initialisiert sein. Hier brauchen wir nur den PALDialog wenn keiner ausgewählt ist.

Reply:
der Methodenname handleInitialize ist evtl. etwas unpassend und hat nichts mit der initialisierung des Wollmux zu tun. OnInitialze macht genau das, den PAL Dialog anzeigen falls kein eintrag in cache.conf.